library/Solarium/Core/Client/Adapter/Guzzle.php

```diff
@@ -74,6 +74,7 @@ public function execute($request, $endpoint)
             RequestOptions::HEADERS => $this->getRequestHeaders($request),
             RequestOptions::BODY => $this->getRequestBody($request),
             RequestOptions::TIMEOUT => $endpoint->getTimeout(),
+            RequestOptions::CONNECT_TIMEOUT => $endpoint->getTimeout()
```

Comment:
Corrected. although as a whole this project doesn't seem valid PSR2 or Symfony Standard so I wasn't able to run phpcs